.github/workflows/msbuild.yml

```diff
@@ -15,26 +15,27 @@ jobs:
         ## Macos jobs tend to stall -- not including them for now
         ## Windows jobs fail on some lit tests and sometimes fail even to
         ## setup a work environment (e.g. dotnet installation)
-        os: [ubuntu-latest]
+        ##os: [ubuntu-latest]
         ##os: [ubuntu-latest, windows-latest, ubuntu-16.04]
         ##os: [ubuntu-latest, macos-latest]
-        ##os: [ubuntu-latest, macos-latest, windows-latest, ubuntu-16.04]
+        os: [
+             ubuntu-latest,
+             macos-latest,
+           ##  windows-latest,
+             ubuntu-16.04,
```

Comment:
Will this have the effect of running everything on Ubuntu twice? Is `ubuntu-latest` different from `ubuntu-16.04`?

Reply:
It verifies that the ubuntu-16.04 build runs properly in ubuntu-18 or eventually ubuntu-20. The tests run in parallel, but if necessary it can be commented out most of the time.